documentation/modules/ROOT/pages/payloads.adoc

```diff
@@ -174,6 +175,46 @@ A Property `errorMessages` with a Collection Characteristic and effective data t
 }
 ----
 
+Another one example for `samm-c:Either`, a Property `speedProperty` has Characteristic and effective data type `samm-c:Either` and the possible values
+`isMoving` (`xref:datatypes.adoc#data-types[xsd:boolean]` data type) or '60' (`Number`) would be serialized in the JSON payload as follows:
+[source,json]
+----
+{
+  "speedProperty": {
+    "left": true
+  }
+}
+
+or
+
+{
+  "speedProperty": {
+    "left": 60
+  }
+}
+----
+
+A Property `speedProperty` has Characteristic and effective data type `samm-c:Either`. Field `samm-c:left` has type `samm-c:Either` and field `samm-c:right` has type `xref:datatypes.adoc#data-types[xsd:boolean]`.
```

Comment:
This is confusing, because it mixes types and Characteristics. I think it would help if the example showed (parts of) an actual Aspect Model.

Reply:
Yes, I will make.

Thanks!